dpgen/analysis/main_func_1.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+from dpdata import LabeledSystem
+
+from dpgen.analysis.analysis_utils import select_logs, process_bar
+
+###########################################################
+def data_collections():
+    while True:
+        print("\n")
+        print("========================================================")
+        print("                    Main Menu 1                        ")
+        print("                                                       ")
+        print(" ( 11)  Collect all data in current dir from OUTCAR file")
+        print("\n Tips: Input q or -10 to exit program                ")
+        print("========================================================")
+        
+        jsel = input("\n Please input the menu index: ")
+        if jsel == str(-10):
+            break
+        elif jsel == str(11):
+            root_dir = input("\n Please input the root dir name: ")
+            str_in   = input("\n Collect (1) all frames (e.g., AIMD task) or (2) final frame (e.g., Optimization task)? ")
+            out_dir  = input("\n Please input the out dir name: ")
```

Comment:
It is highly recommended to use `argparse` not `input` to process input arguments for dpgen commands.

Reply:
> It is highly recommended to use `argparse` not `input` to process input arguments for dpgen commands.

we think it is more conveniet to analysis for users. Users donot need to know the specific argment. Instead, they can use dpgen analysis according to the output the screen.

Reply:
This will 

> > It is highly recommended to use `argparse` not `input` to process input arguments for dpgen commands.
> 
> we think it is more conveniet to analysis for users. Users donot need to know the specific argment. Instead, they can use dpgen analysis according to the output in screen.

This will definitely reduce the flexibility for this function. For a user who's already familiar with the command and wishes to perform a series of tasks, such design will force him/her to manually type command in the interactive mode each time.

Reply:
> > > It is highly recommended to use `argparse` not `input` to process input arguments for dpgen commands.
> > 
> > 
> > we think it is more conveniet to analysis for users. Users donot need to know the specific argment. Instead, they can use dpgen analysis according to the output in screen.
> 
> This will definitely reduce the flexibility for this function. For a user who's already familiar with the command and wishes to perform a series of tasks, such design will force him/her to manually type command in the interactive mode each time.

There's no need to worry about that. For batch analysis, users can prepare an input file containing the input in interactive mode and then perform analysis by redirection such as "dpgen analysis < input > output". For example, Multiwfn (http://sobereva.com/multiwfn/) is the most popular software for wavefunction analysis, , which uses interactive mode. For a analysis tool, it is impossible for users to remember every argument of each function.

Reply:
`dpgen analysis < input` works, but we prefer the argparse for three reasons: 
1. argparse allows arbitrary order of argument.
2. one can easy access the document for ALL arguments. 
3. argparse has type checking.
For the reasons, all other dpgen commands use argparse. Please keep the style of dpgen commands consistent.